src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -164,7 +164,9 @@ default ThreadMember getSelfThreadMember()
      * For this reason, {@link #retrieveThreadMembers()} should be used instead in most cases.
      *
      * <p>The cache this method relies on is empty until JDA sees a member join via a {@link net.dv8tion.jda.api.events.thread.member.ThreadMemberJoinEvent}.
-     * In order for this cache to be updated, the following requirements must be met:
+     * <br>If this thread is spawned from a message sent <i>by this account</i>, then this account will be added to the cache.
```

Comment:
We actually might want to not include this bit.
This functionality is only true because spawning a thread from a message automatically places the author of the original message into the thread. It's basically no different than joining an already created thread.

Either way, the bot would be in the thread and be able to see itself as a member in the cache.

If they left the thread, then the newly added comment here makes it _seem_ like their `getSelfThreadMember` would continue resolving just because the thread was spawned from a message sent by the bot, which is not true.